lib/oli/authoring/media_library/item_options.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Oli.Authoring.MediaLibrary.ItemOptions do
```

Comment:
Really like this pattern for typing and creating structs

Reply:
yeah, i agree